### PR TITLE
Update crate dependencies

### PR DESCRIPTION
**Pull request by tygyh**
# Objective

- Give access to each crate's latest fixes and features.

## Solution

- Update dependencies to latest versions.

### CODE REVIEW

**Comment by valaphee** [comment]
Some crates should have their patch version omitted like
```
async-broadcast = "0.5"
```
should become
```
async-broadcast = "0.6"
```

File: benches/Cargo.toml [hunk 1]
```diff
@@ -7,10 +7,10 @@ publish = false
 license = "MIT OR Apache-2.0"
 
 [dev-dependencies]
-glam = "0.24.1"
+glam = "0.25.0"
```
Comment (alice-i-cecile) [inline benches/Cargo.toml:10]: We should swap to using bevy_math's re-export instead.

File: crates/bevy_mikktspace/Cargo.toml [hunk 2]
```diff
@@ -15,7 +15,7 @@ license = "Zlib AND (MIT OR Apache-2.0)"
 keywords = ["bevy", "3D", "graphics", "algorithm", "tangent"]
 
 [dependencies]
-glam = "0.24.1"
+glam = "0.25.0"
```
Comment (alice-i-cecile) [inline crates/bevy_mikktspace/Cargo.toml:18]: This needs to be synced with bevy_math: is it?

**Comment by mockersf** [comment]
I would prefer to avoid updating to semver compatible version without a justification. the latests versions will already be picked up by cargo anyway, unless someone has a reason not to, which this would block.

I would also prefer to avoid updating unrelated dependencies in a single PR

**Pull request closed without merging**